render_pdf/tex_files/runscript

```diff
@@ -0,0 +1,4 @@
+xelatex template.tex
```

Comment:
There might be an option to suppress these output files. I think this is better in case we end up deleting other files called .out, .log, etc.

Reply:
`xelatex` itself doesn't have an option for cleaning up, but look into invoking it via say [`latexmk`](https://ctan.org/pkg/latexmk?lang=en) (it has `-c` for clean IIRC) or [`arara`](https://ctan.org/pkg/arara): its documentation mentions a directive like

```
% arara: clean: { extensions: [ aux, log ] }
```
— other options I haven't looked at are [`latexrun`](https://github.com/aclements/latexrun) or (further afield) [tectonic](https://tectonic-typesetting.github.io/en-US/).

Reply:
Yup, and I need xelatex for babel/font support. I'll fix this later; I'd rather not waste time now for something that has a clear/simple solution. And it's also not the end of the world if I leave the log files too. I think 
1. we can process a pdf, store it in a bucket to cache, and
2. overwrite it for the next run of render_pdf. 
They're fairly small files, so it's not too large a concern.